src/log.hpp

```diff
@@ -74,6 +74,7 @@ const unsigned max_logs = 8;
 
 enum severity
 {
+    LG_NONE=-1,
```

Comment:
It should now be possible to make this type an `enum class`, right?

Reply:
I'll change this, fix the other issues and push a new version to the same commit

Reply:
> It should now be possible to make this type an `enum class`, right? Although `lg::severity::LG_ERROR` has a bit of redundancy to it…

Depends, remind me which version of the C++ standard Wesnoth is on again?

Reply:
C++17 (mostly)